### PR TITLE
#136/Update issue templates

### PR DESCRIPTION
**Pull request by Hambrsoom**
I added a template for possible improvement issues that we would like to have in the future if we get the chance. Furthermore, I added the environment section in the bug report template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/custom.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-name: 'Improvement Template '
-about:
+name: Improvement template
+about: Serves as a template for improvement issues.
 title: ''
-labels: Improvement
+labels: ''
```
Comment (ferdousara-parvin) [inline .github/ISSUE_TEMPLATE/custom.md]: why did you remove the "improvement" label?

Comment (Hambrsoom) [reply .github/ISSUE_TEMPLATE/custom.md]: sounds like i did it by mistake. i will fix it now
